pkg/cmd/kind/export/logs/logs.go

```diff
@@ -70,14 +73,26 @@ func runE(logger log.Logger, streams cmd.IOStreams, flags *flagpole, args []stri
 
 	// get the optional directory argument, or create a tempdir
 	var dir string
-	if len(args) == 0 {
+	if flags.Out != "" {
+		// prefer the flag
+		dir = flags.Out
+		if len(args) != 0 {
+			cmd.FancyWarn(logger, "Ignoring argument since preferred --out is set. Please stup using an arugment for the export path!")
```

Comment:
probably error instead?

Reply:
typos in `stup` and `arugment`

i guess, if it's deprecated it should continue to work until the deprecation period passed.

Reply:
This would be an error if you set *both*

Reply:
ok, makes sense.

Reply:
instead it's all warnings with a preference chain. I updated it slightly.